perdido/section-styles.js

```diff
@@ -0,0 +1,192 @@
+var blockMar = 30;
+
+window.setupStyles = {
+  // Reset code
+  '*, *:after, *:before': {
+    boxSizing: 'border-box',
+    padding: '0',
```

Comment:
I will publish it, forgot about it. But besides of that 0px when generated on the client is not a problem at all.